### DISCUSSION TITLE
用GoDaddy DNS申请证书失败

### DISCUSSION THREAD

**Discussion by hzgotb** [1Panel-dev/1Panel#6003] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

用 GoDaddy DNS 申请证书失败。日志提示是获取 _came-chanllenge 超时。试了用 curl 在终端获取也是不行，用了代理之后才可以。主要是 api.godaddy.com 无法访问。现在有没办法在请求时使用代理？

### 重现步骤

无

**Comment by zhengkunwang223** [comment]
你是用的 GoDaddy 申请证书吗？ 还是用的 1Panel 的证书模块

**Comment by hzgotb** [comment]
1Panel 的证书模块

**Comment by zhengkunwang223** [comment]
> 1Panel 的证书模块

申请证书时暂时无法支持代理

**Reply by hzgotb** [reply]
http proxy env也不能用吗？我在主机上写了环境变量，用curl是没问题的。
